src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +39,16 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" to={"/" + (query ? `?q=${query}#${searchInputId}` : "")}>
```

Comment:
Well, good remark! I'd like to add a configuration variable, but that would mean updating all the existing links (`<Link to=>` ). Should I create a new issue?